content/en/synthetics/settings/_index.md

```diff
@@ -16,19 +16,18 @@ further_reading:
   text: "Identify Synthetic Bots"
 ---
 
-On the [Synthetic Monitoring settings page][1], you can adjust the following settings:
+On the [Synthetic Monitoring Settings page][1], you can adjust the following settings:
 
 * [Global Variables](#global-variables)
 * [Private Locations][2]
 * [Default settings](#default-settings)
   * [Default Locations](#default-locations)
   * [APM integration for Browser Tests](#apm-integration-for-browser-tests)
 
-Only [Admin and Standard users][3] can access Synthetic Monitoring `Settings` page.
-
 ## Global variables
 
-Variables are global and can be used by multiple [API tests][4] and [browser tests][5]. To create a new global variable, go to the [Global Variables][6] tab of your **Settings** page, and click **New Global Variable** in the upper right corner of your page.
+Global variables can be used in [single][5] and [multistep API tests][link] as well as [browser tests][6] of your test suite. To create a global variable, go to the [Global Variables][7] tab of the **Settings** page, and click **New Global Variable** in the upper right corner.
```

Comment:
@alai97 Looks like we're missing a link for multistep API tests in there!

Reply:
```suggestion
Global variables can be used in [single][5] and [multistep API tests][12] as well as [browser tests][6] of your test suite. To create a global variable, go to the [Global Variables][7] tab in the **Settings** page, and click **New Global Variable** on the upper right corner.
```